guides/common/modules/proc_enrolling-projectserver-in-freeipa-domain.adoc

```diff
@@ -0,0 +1,121 @@
+ifndef::orcharhino[]
++
+For more information, see link:{RHELDocsBaseURL}/8/html-single/installing_identity_management/index#assembly_installing-an-idm-client_installing-identity-management[{RHEL}{nbsp}8 Installing Identity Management] or link:{RHELDocsBaseURL}/9/html-single/installing_identity_management/index#assembly_installing-an-idm-client_installing-identity-management[{RHEL}{nbsp}9 Installing Identity Management].
+endif::[]
++
+[NOTE]
+====
+To install packages on your {ProjectServer}, use the `{foreman-installer}` utility.
+====
+ifdef::foreman-deb[]
++
+. Ensure that the hostname is set to the fully qualified domain name (FQDN); the short name is not sufficient:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# hostname
+{foreman-example-com}
+----
++
+Otherwise, `{foreman-installer}` cannot generate the right principal name that is needed to join the realm.
+endif::[]
+
+.Verification
+* On your {ProjectServer}, check that you are able to resolve a user defined on the {FreeIPA} server.
+For example, to check the `admin` user that {FreeIPA} creates by default:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ id admin
+----
+
+.Enrolling a {ProjectServer} system as a {FreeIPA} client from the command line by using a one-time password
+====
+On the {FreeIPA} server, a user named _admin_ who has administrative privileges on the {FreeIPA} server prepares a host entry for the {ProjectServer} system:
+
+. Authenticate as the {FreeIPA} _admin_ user:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# kinit _admin_
+----
+. Optional: Verify that you have authenticated successfully:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# klist
+----
+. Create a host entry from the command line.
+Specify that you want to use a random password for the enrollment.
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# ipa host-add --random _{project-context}-server.example.com_
+--------------------------------------------------
+ Added host "{project-context}-server.example.com"
+ --------------------------------------------------
+  Host name: {project-context}-server.example.com
+  Random password: W5YpARl=7M.n
+  Password: True
+  Keytab: False
+  Managed by: ipa-server.example.com
+----
+. Enable access to the keytab file by creating a service principal for your {ProjectServer}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# ipa service-add HTTP/_{project-context}-server.example.com_
+----
+
+On the {ProjectServer} system, a user with {Project} administrative privileges enrolls the system into the {FreeIPA} domain:
+
+. Install the {FreeIPA} client packages:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {project-package-install} ipa-client
+----
++
+WARNING: The {project-package-install} command might restart {Project} services during the installation of the package.
```

Comment:
IMHO this only happens on Satellite because the attribute resolves to "foreman-installer install package ..." for satellite and "dnf install" for all other builds.

Reply:
@maximiliankolb You mean I should exclude this for non-satellite builds?

Reply:
I believe so, yes. Unless this is not done by "foreman-maintain" but "dnf needs-restarting"? https://access.redhat.com/solutions/27943

I am not 100% sure. Kindly asking @ekohl

Reply:
Let's just drop the warning. I checked that there are multiple instances of `satellite-maintain packages install` without such a warning. If there is a need to remind users that services will get restarted, it should be handled consistently anyway.